demo/build.gradle

```diff
@@ -54,18 +70,55 @@ android {
 
 // [START maps_android_utils_install_snippet]
 dependencies {
-    implementation 'com.google.maps.android:android-maps-utils:2.2.0'
```

Comment:
I just realized that this line was part of the documentation I believe on https://developers.google.com/maps/documentation/android-sdk/utility/setup.

@arriolac Thoughts on how to handle this when the dependencies are moved to the individual flavors?

Reply:
I'll have to update the snippet on my end once this change has been merged.